aspnetcore/fundamentals/host/hosted-services/samples-snapshot/2.x/BackgroundTasksSample-WebHost/QueuedHostedService.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace BackgroundTasksSample.Services
+{
+    #region snippet1
+    public class QueuedHostedService : IHostedService
```

Comment:
Why are there two copies of QueuedHostedService? For 2.0 and 2.1? The folder structure does not make that obvious.

Reply:
I wish we didn't need to break tradition with this 2.0 and 2.1 coverage. What do you think about dropping the 2.0 coverage entirely? We generally decided not to do it. I was hoping we could craft this without full 2.0 and 2.1 samples, hence my interest in just linking the 2.0 code via snippets. We probably can do it this way if I just identify that the sample app is a 2.1 sample in the topic where the sample is linked.

Reply:
October... https://blogs.msdn.microsoft.com/dotnet/2018/06/20/net-core-2-0-will-reach-end-of-life-on-september-1-2018/
How you want to juggle it in the meantime I'll leave to you and Rick.

Reply:
I vote for 🔪 it now. We already decided internally to *disfavor* 2.0-specific code after 2.1 came out. My understanding is that this decision was based on the difficulty in maintaining the topics+samples. It's just too burdensome to us to maintain 1.x, 2.0, and 2.1 topics/samples.

Let's ask ... that way I won't get into trouble. I'll be next for the 🔪 if I'm not careful! :smile:

@Rick-Anderson TL;DR Do you want to maintain a full, working 2.0 version of the BackgroundTasksSample-WebHost sample because the proposed changes here are only available for 2.1, or do you want to stick with what we decided earlier and cut 2.0 loose for 2.1 scenarios (i.e., drop the 2.0 approach from the topic+sample here)?